includes/class-wc-payments.php

```diff
@@ -976,6 +985,7 @@ public static function ajax_init_platform_checkout() {
 				'store_api_url'     => self::get_store_api_url(),
 				'account_id'        => $account_id,
 			],
+			'user_session'         => isset( $_REQUEST['user_session'] ) ? sanitize_text_field( wp_unslash( $_REQUEST['user_session'] ) ) : null,
```

Comment:
If I understand correctly, if `user_session` is missing, the request will never ever succeed. If so, let's bail early even before making the HTTP request. E.g. like this:

```php
if ( empty( $_REQUEST['user_session'] ) ) {
	wp_send_json_error(
		__( 'User session ID is missing', 'woocommerce-payments' ),
		500
	);
}
```

Reply:
Not exactly. If `user_session` is missing, it simply won't be added in the return URL. Besides, we might need to make some changes in how we handle the `user_session` to be able to redirect existing customers without going through the SMS OTP flow. I'll keep that in mind, though 👍

Reply:
@cesarcosta99 Sorry for the late response here but since this is a call to the platform checkout server to start a session, don't we have to have a `user_session` to authenticate?